Makefile

```diff
@@ -17,6 +17,15 @@ ISO ?= $(BUILD_ROOT)/metalk8s.iso
 CALICO_CNI_PLUGIN_VERSION = 3.5.1
 CALICO_CNI_PLUGIN_BUILD = 1
 
+CONTAINER_IMAGES = \
+	k8s.gcr.io/kube-apiserver@sha256:79b197b6a7334d2ad01ee82a9be82a39bfb50e33bc03263c58c4dfc4e81f71bc \
```

Comment:
For all of these, please have a comment with them saying which tag this is. Also, if we pull-by-digest, do we get the tag in the metadata, i.e. when we import the image, does it show up with tag-name?

Reply:
Ok, will do.
No, there is no tag in the metadata when images are pulled by their digests.

Reply:
So, in the snippet I shared, I pull by tag, but export by digest, in an attempt to have the tags anyway (didn't check yet though *oops*)